README.md

```diff
@@ -269,6 +270,93 @@ return (
 );
 ```
 
+### Enabling DRM protection
+
+Simple streaming of protected assets can be enabled with just a little configuration on `SourceConfig.drmConfig`:
+
+```typescript
+// Source configuration for protected asset.
+const drmSource: SourceConfig = {
+  // Protected stream URL.
+  url:
+    Platform.OS === 'ios'
+      ? 'https://fps.ezdrm.com/demo/video/ezdrm.m3u8' // iOS stream url
+      : 'https://bitmovin-a.akamaihd.net/content/art-of-motion_drm/mpds/11331.mpd', // Android stream url
+  // Stream type.
+  type: Platform.OS === 'ios' ? SourceType.HLS : SourceType.DASH,
+  // DRM setup.
+  drmConfig: {
+    // Stream license URL.
+    licenseUrl:
```

Comment:
did this happen? I only see the 👍 emoji here, no related commits 😅

Reply:
No, since we've started discussing this matter [here](https://github.com/bitmovin/bitmovin-player-react-native/pull/11#discussion_r929237804) I haven't addressed your request just yet. But will soon, no worries 🙂.

Reply:
Here's the commit: c35771fb2595c54c6155ecb71c4a5cddfb7a5939